Gems/Atom/RHI/Vulkan/Code/Source/Platform/Windows/RHI/NsightAftermath_Windows.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <RHI/NsightAftermathGpuCrashTracker_Windows.h>
+
+
+namespace Aftermath
+{
+    void SetAftermathEventMarker( [[maybe_unused]] void* cntxHandle, [[maybe_unused]] const char* markerData, [[maybe_unused]] bool isAftermathInitialized)
+    {
+#if defined(USE_NSIGHT_AFTERMATH)
+        if (isAftermathInitialized)
+        {
+            GFSDK_Aftermath_Result result = GFSDK_Aftermath_SetEventMarker(
+                static_cast<GFSDK_Aftermath_ContextHandle>(cntxHandle), static_cast<const void*>(markerData),
+                static_cast<unsigned int>(strlen(markerData) + 1));
+            AssertOnError(result);
+        }
+#endif
+    }
+
+    void* CreateAftermathContextHandle([[maybe_unused]] ID3D12GraphicsCommandList* commandList, [[maybe_unused]] void* crashTracker)
+    {
+#if defined(USE_NSIGHT_AFTERMATH)
+        GFSDK_Aftermath_ContextHandle aftermathCntHndl = nullptr;
+        // Create an Nsight Aftermath context handle for setting Aftermath event markers in this command list.
+        GFSDK_Aftermath_Result result = GFSDK_Aftermath_DX12_CreateContextHandle(commandList, &aftermathCntHndl);
```

Comment:
Name of the function has DX12 in it. Is this accurate? Were you able to test this code by simulating a gpu crash and seeing if Aftermath catches it and creates a dump log?

Reply:
We did this change because of a GPU crash occurring on Vulkan, so I can confidently says that it works. However you're right this function makes no sense here, and is not even called.

Reply:
In fact this whole file is a leftover that was not even compiled, I removed it.